opendit/models/latte/latte_t2v.py

```diff
@@ -1138,6 +1146,29 @@ def forward(
             If `return_dict` is True, an [`~models.transformer_2d.Transformer2DModelOutput`] is returned, otherwise a
             `tuple` where the first element is the sample tensor.
         """
+
+        # 0. Split batch for data parallelism
+        if get_data_parallel_size() > 1:
```

Comment:
maybe there is no need to determine who is conditional by:
1. set dp size = 2
2. xxx = split_sequence(xxx, group=dp_group)

Reply:
Fixed.